bokehjs/src/coffee/widget/vbox.coffee

```diff
@@ -35,11 +40,15 @@ class VBox extends BaseBox.Model
   defaults: ->
     return _.extend {}, super(), {
       children: []
+      css_classes: []
```

Comment:
You don't have to override this. Unless inheritance chain is broken, then it should be fixed instead.